cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -0,0 +1,22 @@
+[id="understanding-odo"]
+= Understanding OpenShift Do
+include::modules/openshift-developer-cli-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: understanding-odo
+toc::[]
+
+{odo-title-full} ({odo-title}) is a fast and easy-to-use CLI tool for creating applications on {product-title}. {odo-title} allows developers to concentrate on creating applications without the need to administrate an {product-title} cluster itself.
+Creating deployment configurations, build configurations, service routes and other {product-title} elements are all automated by {odo-title}.
+
+Existing tools such as `oc` are more operations-focused and require a deep understanding of Kubernetes and OpenShift concepts. {odo-title} abstracts away complex Kubernetes and OpenShift concepts allowing developers to focus on what is most important to them: code.
+
+
+== Key features
+
+{odo-title} is designed to be simple and concise with the following key features:
+
+* Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
+* Completely client based. No server is required within the {product-title} cluster for deployment.
+* Supports many languages and frameworks such as Node.js, Java, Ruby, Perl, PHP, and Python.
```

Comment:
Officially supported languages are just Node.JS and Java. The rest can't be currently guaranteed :-(

I'm not sure how properly formulate this

Reply:
Good catch !!!